talisker/flask.py

```diff
@@ -49,10 +49,22 @@ def sentry(app, dsn=None, transport=None, **kwargs):
     return sentry
 
 
+def add_view_name(response):
+    try:
+        endpoint = flask.request.endpoint
+        module = flask.current_app.view_functions[endpoint].__module__
+        response.headers['X-View-Name'] = module + '.' + endpoint
```

Comment:
let's move this to a (new) `else:` clause, since it's not going to trigger an exception and is about handling the results of potential exception-raising code.